src/teleop/gui/src/store/modules/erd.js

```diff
@@ -0,0 +1,64 @@
+// initial state
+const state = {
+    route: [],
+    waypointList: [],
+    highlightedWaypoint: -1,
+    autonEnabled: false,
+    teleopEnabled: true,
+    odomFormat: "DM",
+    clickPoint: {
+      lat: 0,
+      lon: 0
+    },
+  
+  }
+  
+  // getters
+  const getters = {
+    route: state => state.route,
+    waypointList: state => state.waypointList,
+    highlightedWaypoint: state => state.highlightedWaypoint,
+    autonEnabled: state => state.autonEnabled,
+    teleopEnabled: state => state.teleopEnabled,
```

Comment:
Delete auton and teleop

Reply:
Oh didn't realize you'd already said this, nice